sequencing.go

```diff
@@ -45,3 +53,37 @@ type Hash = []byte
 type Batch struct {
 	Transactions []Tx
 }
+
+// SubmitRollupTransactionRequest is a request to submit a transaction from rollup to sequencer
+type SubmitRollupTransactionRequest struct {
+	RollupId RollupId
+	Tx       Tx
+}
+
+// SubmitRollupTransactionResponse is a response to submitting a transaction from rollup to sequencer
+type SubmitRollupTransactionResponse struct {
```

Comment:
Should this at least have an error type for the response? 
Or is this intentionally left blank to be implemented by the specific sequence implementation?